src/main/java/teammates/common/util/Const.java

```diff
@@ -380,19 +380,36 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
     }
     
     public static class FeedbackQuestion {
-    
+
+        public static final String QUESTION_INDEX = "${qnIdx}";
+        public static final String RESPONSE_INDEX = "${responseIdx}";
+        public static final String DISABLED = "${disabled}";
+        public static final String FEEDBACK_RESPONSE_TEXT = "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}";
+        public static final String CHECKED = "${checked}";
+        public static final String OPTION_RECIPIENT_DISPLAY_NAME = "${optionRecipientDisplayName}";
```

Comment:
@kanghj these Consts are used in each "Questiontype" files. Can we define these consts here ?